docker-compose.yaml

```diff
@@ -0,0 +1,23 @@
+services:
+  medusa-proxy:
+    image: datawookie/medusa-proxy
+    build: . #if need build from local Dockerfile
+
+    environment:
+      # - HEADS=2
+      # - TORS=5
+      # - TOR_EXITNODES={de}
+      - TOR_USEBRIDGES=1
+      # - TOR_BRIDGES='Bridge obfs4 148.252.111.70:9040 F4462199499EF2D3E9F7A686D8AE71C50E1CB5E0 cert=57ypxWpn24PhWqKbUAqYRhIXX7uPk1Uo6m/mslymxcl6D6yTb6CjoLIJ96IfwIGOXXa+cw iat-mode=0,
+      #     Bridge obfs4 37.18.133.75:44821 D40DA77CA68F39666F77CE8BA6FF332BF8DB3F31 cert=B4yVW8heE83luCJt+oQN1kDB/j4kWkNx6mtOc9O6GhLAV8zJx0lfUI6zWO9UxUoV5PX/Zw iat-mode=0,
+      #     Bridge obfs4 51.81.26.157:443 8A7322A463C051DB6DC35B1159F119FC3373BB06 cert=kp6Czj/f+McG9OKwltQ4kGb41mjj8Mzp3flpTG8/VK5zXtfnZ+DToe33fumyq7Yq7WnbGA iat-mode=0
+
+    container_name: medusa-proxy
+    volumes:
+      - ./templates/bridges.lst:/templates/bridges.lst:ro
```

Comment:
Since the bridges file is not really a template I'd prefer not to mount it into the templates folder. How about either an absolute path on the container:

```
    volumes:
      - ./bridges.lst:/bridges.lst:ro
```

or a relative path

```
    volumes:
      - ./bridges.lst:bridges.lst:ro
```